backend/cms/models/configuration.py

```diff
@@ -0,0 +1,8 @@
+from django.db import models
+
+
+class Configuration(models.Model):
```

Comment:
@timoludwig do you have any opinion on this model?

Reply:
I think it has a very limited use case (as we store all non-sensitive config in the settings.py), but for storing keys it's fine.